.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+## Description
```

Comment:
Ooh, you added PR templates? So PRs will auto populate with these now? Awesome.

Reply:
Yes, when you open a PR it will automatically have this in the text field